src/libraries/System.Console/tests/ManualTests/ManualTests.cs

```diff
@@ -138,8 +138,6 @@ public static IEnumerable<object[]> GetKeyChords()
             yield return MkConsoleKeyInfo('\x02', ConsoleKey.B, ConsoleModifiers.Control);
             yield return MkConsoleKeyInfo(OperatingSystem.IsWindows() ? '\x00' : '\x02', ConsoleKey.B, ConsoleModifiers.Control | ConsoleModifiers.Alt);
             yield return MkConsoleKeyInfo('\r', ConsoleKey.Enter, (ConsoleModifiers)0);
-            // windows will report '\n' as 'Ctrl+Enter', which is typically not picked up by Unix terminals
-            yield return MkConsoleKeyInfo('\n', OperatingSystem.IsWindows() ? ConsoleKey.Enter : ConsoleKey.J, ConsoleModifiers.Control);
```

Comment:
Can we keep a "fixed" test here?

Reply:
You mean keep the test for windows only?

Reply:
I mean continue to validate this behavior on Windows, and have the test validate on Linux/macOS whatever the behavior is there now.

Reply:
Done.